workflows/metatdenovo.nf

```diff
@@ -155,15 +156,23 @@ workflow METATDENOVO {
     }
 
     //
-    // MODULE: Run Megahit on all interleaved fastq files
+    // MODULE: Run Megahit or RNAspades on all interleaved fastq files
     //
-    MEGAHIT_INTERLEAVED(
-        ch_pe_reads_to_assembly.collect(),
-        ch_se_reads_to_assembly.collect(),
-        'all_samples'
-    )
-    ch_assembly_contigs = MEGAHIT_INTERLEAVED.out.contigs
-    ch_versions = ch_versions.mix(MEGAHIT_INTERLEAVED.out.versions)
+    if ( params.rnaspades ) {
+        SPADES( SEQTK_MERGEPE.out.reads.collect(),
+                'all_samples'
+        )
+       ch_assembly_contigs = SPADES.out.transcripts
+       ch_versions = ch_versions.mix(SPADES.out.versions)
+    } else {
+        MEGAHIT_INTERLEAVED(
+            ch_pe_reads_to_assembly.collect(),
+            ch_se_reads_to_assembly.collect(),
+            'all_samples'
+        )
+        ch_assembly_contigs = MEGAHIT_INTERLEAVED.out.contigs
+        ch_versions = ch_versions.mix(MEGAHIT_INTERLEAVED.out.versions)
+    }
```

Comment:
I'd prefer a parameter `assembler` that takes "megahit" or "rnaspades" as values, with the former as default.